hackathon_site/hardware/serializers.py

```diff
@@ -89,6 +94,20 @@ def get_team_code(obj: Order):
         return obj.team.team_code
 
 
+class IncidentCreateSerializer(IncidentListSerializer):
```

Comment:
I think in this case, you don't want to subclass `IncidentListSerializer`, because we don't want the nested `order_item` serializer. Looking at the request body in django:

```
{
  "state": "Heavily Used",
  "time_occurred": "2021-11-19T00:56:12.694Z",
  "description": "string",
  "order_item": {
    "hardware": 0,
    "order": 0,
    "part_returned_health": "Healthy"
  }
}
```

We would have to specify the full `order_item` in the request (according to swagger anyway), which is unnecessary. It would be better if `order_item` were just the OrderItem id. 

This is contrary to my [comment](https://github.com/ieeeuoft/hackathon-template/pull/312/files#r752782211) on #312 where we can subclass, because the nested serializer field doesn't need to be part of the request.

Reply:
yep